charts/spinnaker/values.yaml

```
@@ -360,6 +360,28 @@ rbac:
   # Specifies whether PSP resources should be created
   pspEnabled: false
 
+pdb:
+  # Specifies whether a PodDisruptionBudget should be created
+  #
```

Nitpick and elsewhere. 
```suggestion
  # Specifies whether a PodDisruptionBudget should be created
```

---

👍🏻